### ISSUE TITLE
FCA Enhanced Diagnostics?

### ISSUE THREAD

**Issue by black-jmyntrn** [PowerBroker2/ELMduino#266]
The enhanced diagnostics allows the read and clear(1) trouble codes from proprietary modules such as ABS, Airbag, Transmission, Body Control, and many others. You also get access to hundreds of parameters and sensors that are not available over standard OBD2.

I have an ESP32 based PCB board and wondering if ELMduino can access the enhanced diagnostics.

**Comment by jimwhitelaw** [comment]
Yes, ELMduino can access enhanced diagnostics using custom headers and PIDs. See this [example program](https://github.com/PowerBroker2/ELMduino/blob/master/examples/ESP32_CustomHeader/ESP32_CustomHeader.ino).
